apps/google-analytics-4/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3.8'
+
+services:
+  dynamodb:
+    image: amazon/dynamodb-local
+    restart: always
+    volumes:
+      - .:/home/dynamodblocal/data
+    ports:
+      - 8000:8000
+    command: '-jar DynamoDBLocal.jar -sharedDb -dbPath /home/dynamodblocal/data/'
+
+  dynamodb-admin:
+    image: aaronshaf/dynamodb-admin
```

Comment:
😨 using some-dude-made-it packages like this... maybe not ideal? do we _need_ this?

Reply:
or rather _what would be our backup plan if this were to disappear from github or become abondonware with security vulns_

Reply:
btw in general agree with sentiment of not adding unnecessary dependencies, but we've used this package elsewhere purely for local development / exploration and it's pretty good. Repo seems fairly active and well supported https://github.com/aaronshaf/dynamodb-admin

Reply:
Im not married to it, I pulled it from somewhere I saw it but don't see this in the `slack` app come to think of it 🤔

It was convenient to explore visually, but can also use the aws cli for verification and reduce dependencies/context switching for folks